content/articles/Integrating-YouTube-videos-into-a-Django-app/index.md

```diff
@@ -0,0 +1,309 @@
+﻿### Introduction
+
+Modern Organisations, bloggers, companies, and brands are featuring both pictures and videos on their websites to strengthen their content and to also promote their YouTube channels.
+
+It is very convenient because using YouTube unlike directly uploading a video, saves you space, cost, and compatibility issues. Django being on the rise, it is important to know how this can be achieved using Django.
+
+We will create a simple blogging app and embed youtube videos in it so that when we write a blog, we can add a youtube video to it.
+
+### Prerequisites
+
+1.  Install and set up a programming environment for python 3
+2.  Have some fundamental knowledge in python and the Django framework
+
+#### Step 1: Setting up the environment and downloading required packages
+
+We will begin by creating a directory that will contain our project.
+Open the command line and type in the following:
+
+```bash
+mkdir projectX
+```
+
+Change your current directory to projectX:
```

Comment:
Kindly add some sentences or paragraphs to explain some of the steps like this one.